backend/src/Edna/Dashboard/DB/Query.hs

```diff
@@ -77,6 +86,24 @@ deleteSubExperiment (SqlId subExpId) =
      )
   ) seSubExperimentId
 
+-- | Create a new sub-experiment as a sibling of existing one (sharing
+-- the same experiment ID).
+cloneSubExperiment :: SubExperimentId -> Text -> Params4PL -> Edna SubExperimentRec
+cloneSubExperiment sqlId newName result = do
+  experimentId <- getExperimentId sqlId
```

Comment:
So, here and in `getMeasurements` I think `experimentId` should a function parameter.

Reply:
Ok, I'll try, doesn't sound hard, but let me see.